tools/exporter/csvExporter.py

```diff
@@ -114,9 +205,12 @@ def save_converted_audit_to_file(self, output_csv_path, allow_overwrite):
 
         :param output_csv_path:  The full path to the file to save
         """
-        if os.path.isfile(output_csv_path) and not allow_overwrite:
-            sys.exit('File already exists at ' + output_csv_path + '\nPlease set allow_overwrite to True in confil.yaml file. See ReadMe.md for further instruction')
-        elif os.path.isfile(output_csv_path) and allow_overwrite:
+        file_exists = os.path.isfile(output_csv_path)
+        if file_exists and not allow_overwrite:
+            sys.exit(
```

Comment:
sys.exit() is a bit harsh here. I suggest simply skip saving the file if it exists.